data-raw/R2R_TMH_habitat_inputs/documentation/inchannel_rearing_process_and_Review.Rmd

```diff
@@ -0,0 +1,208 @@
+
+## Objectives
+
+**Objective 1:** Is it possible to use a coarse estimate of gradient by dividing the reach length by the maximum and minimum elevation difference?
+
+**Objective 2:** Calculate the total proportion suitable for North and Middle Yuba Rivers. Use this value and each reaches gradient class to determine an in-channel rearing proportion suitable that can be applied to other reaches within the same gradient class.
+
+```{r include=FALSE}
+# tables from the STillwater Sciences Report 
+table_f_1 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table F-1") |> janitor::clean_names() 
+
+knitr::kable(table_f_1, caption = "Table F-1")
+
+table_f_6 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table F-6") |> janitor::clean_names() 
+
+knitr::kable(table_f_6, caption = "Table F-6")
+
+
+table_4_1 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table 4-1") |> janitor::clean_names() 
+
+knitr::kable(table_4_1, caption = "Table 4-1")
+
+
+```
+
+## Methods
+
+### Objective 1 - coarse vs. fine elevation estimate comparison
+
+As channels steepen, the ratio of suitable salmonid in-channel habitat (pools, riffles, and runs) lessens. The percent of a channel suitable for in-channel rearing depends on how steep the reach is. A coarse elevation profile for each reach was calculated by dividing the elevation difference from top to bottom by the reach length. This is a much simpler approach than calculating the gradient with any finer granularity. A finer approach was used on Middle and North Yuba River to determine if the coarse estimate would be sufficient for this analysis.
+
+#### Coarse gradient estimate
+
+Estimate the elevation of the North and Middle Yuba Rivers by dividing the river length by the elevation difference of the highest and lowest points. The elevations were acquired using the reach definitions provided in Table 4-1.
+
+$coarseGradient = (elevationHigh - elevationLow)/riverLength * 100$
+
+```{r include=FALSE}
+
+coarse_elevations <- data.frame('subbasin' = c("NY", "MY"), 
+                         'elevation_low_ft' = c(1930, 1178), 
+                         'elevation_high_ft' = c(4545, 4640)
+                         ) |> 
+  left_join(table_4_1) |> 
+  mutate(length_ft = upstream_rm * 5280,
+         gradient = (elevation_high_ft - elevation_low_ft)/length_ft * 100,
+         channel_gradient = ifelse(gradient < 1, "0-1%",
+                                    ifelse(gradient > 1 & gradient < 2, "1–2%",
+                                           ifelse(gradient > 2 & gradient < 4, "2–4%",
+                                                  ifelse(gradient > 4 & gradient < 8, "4–8%",
+                                                         ifelse(gradient > 8, "8-12%",
+                                                                "uhoh"))))))
```

Comment:
what is the uhoh?

Reply:
Ha. An `ifelse` has to end in something. It's just a placeholder to make sure all the bases are covered. Can change to `case_when`.